composer.json

```diff
@@ -13,15 +13,15 @@
         }
     ],
     "require": {
-        "php": "^7.1 | ^8.0",
+        "php": "^7.2 | ^8.0",
```

Comment:
not related to this PR

Reply:
Bumping to 7.2 was done to be able to remove parameter type declaration from write method. Check PR description for more info :)

Reply:
I get that removing the support of 7.1 ease the PR (even if there are ways to workarround).
Still, this is not related to this PR.

Reply:
I think it was done to make CI green. Would you prefer to remove it and let CI red for this PR?

Reply:
Well, if you guys find a way to make it run under PHP 7.1 :man_shrugging: I've tried everything I had in mind, but I couldn't find a way without locking maximum Monolog version... Keep in mind that releasing this as-is with PHP 7.1 support **WILL** break projects that depend on the bundle and don't have a locked version for it.